packages/theme-patternfly-org/templates/mdx.js

```diff
@@ -20,42 +22,75 @@ const InlineAlert = ({
   </Alert>
 );
 
+const sourceOrder = {
+  react: 1,
+  'design-guidelines': 2
+};
+const defaultOrder = 99;
+
+const sortSources = (s1, s2) => {
+  const s1Index = sourceOrder[s1] || defaultOrder;
+  const s2Index = sourceOrder[s2] || defaultOrder;
+  if (s1Index === defaultOrder && s2Index === defaultOrder) {
+    return s1.localeCompare(s2);
+  }
+
+  return s1Index > s2Index ? 1 : -1;
+}
+  
+
 export const MDXTemplate = ({
   id,
   designSnippet,
   sources = {},
   path,
   children
 }) => {
-  const isSinglePage = Object.keys(sources).length === 1;
+  const sourceKeys = Object.keys(sources).sort(sortSources);
+  const isSinglePage = sourceKeys.length === 1;
+  const { pathname } = useLocation();
+  let activeSource = pathname.split('/').pop();
+  if (!sourceKeys.includes(activeSource)) {
+    activeSource = 'react';
+  }
 
   return (
     <React.Fragment>
       <SkipToContent href="#main-content">Skip to Content</SkipToContent>
       <SideNavLayout>
-        <PageSection id="main-content" className="ws-section">
+        <PageSection
+          id={isSinglePage ? 'main-content' : 'nav-content'}
+          className="ws-first-section"
+          type={isSinglePage ? 'default' : 'nav'}
+        >
           <Title size="4xl" headingLevel="h1" className="ws-page-title">
             {id}
           </Title>
           {designSnippet && React.createElement(designSnippet.Component)}
           {!isSinglePage && (
-            <ul>
-              {Object.keys(sources)
-                .map(source => (
-                  <li key={source}>
+            <Tabs activeKey={activeSource} className="ws-source-tabs">
+              {sourceKeys.map(source => (
+                <Tab
+                  key={source}
+                  eventKey={source}
+                  title={(
                     <Link to={source}>
-                      {source}
+                      {capitalize(source.replace(/-/g, ' '))}
```

Comment:
Currently users can only click directly on the link within the Tab, rather than anywhere on the Tab (including the padding), to select that Tab.

Reply:
Fixed!